### PR TITLE
Add a README.

### PR DESCRIPTION
**Pull request by Esper89**
I figured it would be a good idea to have info on how to build the modloader on Linux available *outside* the Discord server, so I wrote a README.

### CODE REVIEW

**Comment by Esper89** [comment]
Wow, I'm impressed that adding one markdown file made the CI fail. 🤣

**Comment by Esper89** [comment]
There we go. 😌

**Review by konsti219 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+### Windows
+
+Download the modloader (`astro_modloader.exe`) from the [releases
+page](https://github.com/AstroTechies/astro_modloader/releases/latest), below the changelog.
+
+### Linux
+
+Pre-built binaries are not yet dsitributed for Linux. To build the modloader yourself on Linux:
+
+- Use your distribution's package manager to install `rustup`, `git`, and `build-essential`,
```
Comment (localcc) [inline README.md:16]: maybe make a note that `build-essential` is distro specific?

**Review by localcc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Esper89) [reply README.md]: Done! I also added a note that `build-essential` might be installed by `rustup`.

**Comment by CJendantix** [comment]
You can have temporary environment variables that only last for one command, rather than staying in the environment, by prefixing the commands with the variables rather than exporting them beforehand. I recommend changing
```
export USE_PRECOMPILED_CPP_LOADER=1
export USE_PREBUILT_ASSETS=1
cargo build --release
```
to `USE_PRECOMPILED_CPP_LOADER=1 export USE_PREBUILT_ASSETS=1 cargo build --release`

**Review by localcc — APPROVED** [review]

**Pull request merged**